timely/src/progress/subgraph.rs

```diff
@@ -165,7 +169,19 @@ where
             builder.add_edge(source, target);
         }
 
-        let (tracker, scope_summary) = builder.build();
+        let (mut tracker, scope_summary) = builder.build();
+
+        let path = self.path.clone();
+        tracker.tracker_logger = worker.log_register().get("timely/tracker").map(|logger| (path.clone(), logger));
+
+        // Perhaps log information about the creation of subgraph.
+        if let Some(l) = self.logging.as_mut() {
+            l.log(crate::logging::SubgraphEvent{
+                id: worker.index(),
+                addr: path.clone(),
+                timestamp_type: std::any::type_name::<TInner>().to_string(),
+            });
+        }
```

Comment:
I'd love to reframe this as a `TrackerEvent` and have it be part of the line just up above (i.e. "tracker came in to existence". I suspect something like `Tracker::install_logger(...)` could do both of those things and wrap up the abstraction well. I'm happy to do that after the fact if that works for you.